### PR TITLE
[Field Monitoring] optimize api usage

### PR DESCRIPTION
**Pull request by roman-karpovich**
there are several slow endpoints with response time >1s

### CODE REVIEW

**Comment by roman-karpovich** [comment]
[ch17942]

**Comment by robertavram** [comment]
lgtm , let me label ready when ready.

**Comment by roman-karpovich** [comment]
ok. it would be good to wait a bit for frontend to be sure all required
fields are in place

чт, 6 февр. 2020 г., 22:45 Robert Avram <notifications@github.com>:

**Comment by robertavram** [comment]
still holding on this?

**Comment by roman-karpovich** [comment]
this is not implemented on frontend side yet. now main priority is to finish data collection structure migration, so still waiting

**Comment by robertavram** [comment]
@roman-karpovich  
conflicting with develop.

File: src/etools/applications/field_monitoring/planning/views.py [hunk 1]
```diff
@@ -267,6 +268,17 @@ class InterventionsViewSet(
     serializer_class = InterventionWithLinkedInstancesSerializer
 
 
+class PartnersViewSet(
+    FMBaseViewSet,
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = PartnerOrganization.objects.filter(
+        blocked=False, deleted_flag=False, manually_blocked=False, hidden=False,
```
Comment (robertavram) [inline src/etools/applications/field_monitoring/planning/views.py:277]: name_isnull false,
deleted_flag=False

Comment (roman-karpovich) [reply src/etools/applications/field_monitoring/planning/views.py]: updated

**Pull request merged**